package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.1.0",
   "private": true,
   "license": "BSD-3-Clause",
+  "homepage": "http://oddbird.github.io/Metecho",
```

Comment:
Is this required? I'm not sure we really want this, since it's the URL of the dev deployment of the component library -- not the production app itself.

Reply:
This was a gh-pages requirement, I think. The docs for `storybook-deployer` don't mention that this is required.